src/concurrency.md

```diff
@@ -33,6 +34,40 @@ fn main() {
 }
 ```
 
+[ex-rayon-any-all]: #ex-rayon-any-all
+<a name="ex-rayon-any-all"></a>
+## Check the elements of an array against a predicate in parallel
+
+[![rayon-badge]][rayon] [![cat-concurrency-badge]][cat-concurrency]
+
+The `rayon` crate provides a [`ParallelIterator`] trait which includes parallelized implementations of many of the methods from the standard library's [`Iterator`] trait.
```

Comment:
I'd suggest to omit this first paragraph as the reader should already be familiar with rayon basics via the previous examples. On the other hand we might want to mention that `rayon::any` and `rayon::all` are parallel versions of their std counterparts.

Reply:
👍 I can mention the std counterparts in the second paragraph.